MvvmCross-Forms/MvvmCross.Forms/Views/MvxFormsPagePresenter.cs

```diff
@@ -721,4 +775,73 @@ public virtual void ReplacePageRoot(Page existingPage, Page page, MvxPagePresent
             }
         }
     }
+
+    public static class NavigationHierarchyHelper
```

Comment:
This needs to be moved into it's own file

Reply:
Yep, any suggestions on where it should go - it's a pretty generic helper that doesn't have to be internal to MVX but is specific to XF

Reply:
I think it can just stay in the MvvmCross.Forms package. Just give it a Mvx name and we can ship it.